### PR TITLE
Implement a retry button for when contributors load failed

### PR DESCRIPTION
**Pull request by ogapants**
## Issue
none

## Overview (Required)
Loading will result in an error if you do not have a cache and can't connect to the Internet, but nothing changes on the screen.
In that case, I think that the application needs an approach such as showing a retry button.

I implemented it on ContributorsFragment for the time being, I think that the similar processing is necessary on other screens.
But since they are numerous, we will create an Issue if necessary 🙇 

## Links
none

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

**Comment by ogapants** [comment]
I thought now, should I show a Snackbar with retry action? 🤔

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
     <string name="map">Map</string>
     <string name="information">Information</string>
     <string name="settings">Settings</string>
+    <string name="retry">retry</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: Can you change this to `Retry`?

File: app/src/main/res/layout/fragment_contributors.xml [hunk 2]
```diff
@@ -33,20 +33,22 @@
...
 
-        </FrameLayout>
+        <Button
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:onClick="@{viewModel::onClickRetry}"
+            android:text="@string/retry"
+            android:textColor="@color/white"
+            android:visibility="@{viewModel.retryVisibility}"
```
Comment (konifar) [inline app/src/main/res/layout/fragment_contributors.xml]: Can you use `@style/Button`?

```java
<Button
    style="@style/Button"
    android:layout_gravity="center"
    android:onClick="@{viewModel::onClickRetry}"
    android:text="@string/retry"
    android:visibility="@{viewModel.retryVisibility}"
```

**Comment by konifar** [comment]
> I thought now, should I show a Snackbar with retry action? 🤔

Hmm, I don't know which is better 🤔 But I prefer Snackbar.
Can you try? 🙇

**Comment by ogapants** [comment]
Okay! I'll try it 💪

**Comment by ogapants** [comment]
How about this? I feel that the code is simpler 🤔

**Comment by konifar** [comment]
Looks good. I have one more order. Can you change the button text color? to white?

**Comment by ogapants** [comment]
Yes, sir.
BTW, Should I move the error message to strings.xml?

**Comment by konifar** [comment]
> Should I move the error message to strings.xml?

Yes, please!

**Comment by ogapants** [comment]
I did it 😃

**Comment by konifar** [comment]
LGTM! Thanks 👍

**Pull request merged**